notebook/static/notebook/js/actions.js

```diff
@@ -423,6 +415,21 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
+        'toggle-toolbar':{
+            help: 'hide/show the toolbar',
+            handler : function(env){
+                $('div#maintoolbar').toggle();
```

Comment:
I didn't add the jQuery here, I think you did ;-) 

I was simply moving this block into another place in the file near other shortcuts like it.